build/windows/installer/global.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sdk": {
+        "version": "7.0.102",
```

Comment:
Is this the latest dotnet version? Is this somehow tied to the wix toolset's version?

Reply:
It's a fairly arbitrary minimum version for the CI.

Reply:
I guess the version is tied to the wix toolset to the extend that wix has a minimal version requirement. According to https://wixtoolset.org/docs/intro/ that is version 6 or later for wix v4, so 7 seems perfectly fine :)